src/GraphQL/Types/Composite/AutoRegisteringInputObjectGraphType.cs

```diff
@@ -32,12 +32,20 @@ public AutoRegisteringInputObjectGraphType(params Expression<Func<TSourceType, o
         {
             _excludedProperties = excludedProperties;
             Name = typeof(TSourceType).GraphQLName();
+            ConfigureGraph();
+            AutoRegisteringHelper.ApplyGraphQLAttributes<TSourceType>(this);
             foreach (var fieldType in ProvideFields())
             {
                 _ = AddField(fieldType);
             }
         }
 
+        /// <summary>
+        /// Applies default configuration settings to this graph type prior to applying <see cref="GraphQLAttribute"/> attributes.
+        /// Allows the ability to override the default naming convention used by this class without affecting attributes applied directly to this class.
+        /// </summary>
+        protected virtual void ConfigureGraph() { }
```

Comment:
Before or after `AutoRegisteringHelper.ApplyGraphQLAttributes<TSourceType>(this);` ?

Reply:
The method exists solely for derived classes to run before the attributes are applied, such as setting a default GraphQL name (that is overridable with `[Name]`).  Derived classes can use their constructor to run code after the attributes are applied.  I think I listed as such in the migration docs.

Reply:
ok